tests/unit/data/darwin/Library/Application Support/Firefox/Profiles/asdflulz.Release/extensions/fxos_1_3_simulator@mozilla.org/b2g/B2G.app/Contents/MacOS/b2g-bin

```diff
@@ -0,0 +1 @@
+Shh, I am pretending to be a simulator.
```

Comment:
They are the files pretending to be simulators that the code finds as part of the tests.